From f33c9b7515e7da449ea9911623ea9ad3f73181b4 Mon Sep 17 00:00:00 2001
From: d2weber <d2weber@noreply.codeberg.org>
Date: Fri, 7 Nov 2025 10:36:01 +0100
Subject: [PATCH] Add categories and keywords to .desktop file (#111)

We need "Categories" in the .desktop file to pass the `flatpak-builder-lint repo`

Reviewed-on: https://codeberg.org/lk108/deltatouch/pulls/111
Co-authored-by: d2weber <d2weber@noreply.codeberg.org>
Co-committed-by: d2weber <d2weber@noreply.codeberg.org>
---
 deltatouch.desktop.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/deltatouch.desktop.in b/deltatouch.desktop.in
index 029d039..23b2137 100644
--- a/deltatouch.desktop.in
+++ b/deltatouch.desktop.in
@@ -4,7 +4,9 @@ Exec=deltatouch %U
 Icon=deltatouch.svg
 Terminal=false
 Type=Application
+Categories=Network;Chat;InstantMessaging;
 X-Lomiri-Touch=true
 X-Purism-FormFactor=Workstation;Mobile;
 MimeType=application/x-webxdc;x-scheme-handler/openpgp4fpr;x-scheme-handler/dcaccount;x-scheme-handler/dclogin;x-scheme-handler/mailto;
+Keywords=dc;chat;delta;messaging;messenger;email;webxdc
 X-GNOME-UsesNotifications=true
